src/ports/squids/component.ts

```diff
@@ -0,0 +1,172 @@
+import { ISquidComponent, Squid, SquidMetric } from "./types";
+import {
+  ECSClient,
+  ListServicesCommand,
+  ListTasksCommand,
+  DescribeTasksCommand,
+  ListServicesRequest,
+  UpdateServiceCommand,
+  DescribeServicesCommand,
+} from "@aws-sdk/client-ecs";
+import { IFetchComponent } from "@well-known-components/interfaces";
+import { IPgComponent } from "@well-known-components/pg-component";
+import { getPromoteQuery } from "./queries";
+import { getMetricValue, getSquidsNetworksMapping } from "./utilts";
+import { Network } from "@dcl/schemas";
+
+const AWS_REGION = "us-east-1";
```

Comment:
is this parameter mandatory? user should already have this region assigned as default.

Reply:
yes, otherwise it throws: `Error listing services: Error: Region is missing` when initializing the SDK

Reply:
ok